conf.yaml

```
@@ -595,8 +595,8 @@ contents:
               - title:      Rust Client
                 prefix:     rust-api
                 current:    master
```

I think this also needs to be `main`
```suggestion
                current:    main
```